roles/kubernetes-apps/external_cloud_controller/openstack/templates/external-openstack-cloud-controller-manager-ds.yml.j2

```diff
@@ -36,7 +36,7 @@ spec:
       serviceAccountName: cloud-controller-manager
       containers:
         - name: openstack-cloud-controller-manager
-          image: {{ docker_image_repo }}/k8scloudprovider/openstack-cloud-controller-manager:{{ external_openstack_cloud_controller_image_tag }}
+          image: registry.k8s.io/provider-os/openstack-cloud-controller-manager:{{ external_openstack_cloud_controller_image_tag }}
```

Comment:
Could you add a new var for this in `kubespray-defaults/defaults/main/download.yml` and use that image here?